vespajlib/src/main/java/com/yahoo/geo/DistanceParser.java

```diff
@@ -0,0 +1,58 @@
+// Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+package com.yahoo.geo;
+
+public class DistanceParser {
+    // according to wikipedia:
+    // Earth's equatorial radius = 6378137 meter - not used
+    // meters per mile = 1609.344
+    // 180 degrees equals one half diameter equals PI*r
+    // Earth's polar radius = 6356752 meter
+
+    public final static double  m2deg =            180.0 / (Math.PI * 6356752.0);
+    public final static double km2deg = 1000.000 * 180.0 / (Math.PI * 6356752.0);
+    public final static double mi2deg = 1609.344 * 180.0 / (Math.PI * 6356752.0);
+
+    public final double degrees;
```

Comment:
Consider keeping this private and add accessor instead.

Reply:
i'm actually considering adding a "static double parse(String)" instead, that feels more natural to me.